GiniComponents/GiniInternalPaymentSDK/Sources/GiniInternalPaymentSDK/PaymentReview/PaymentReviewModel.swift

```diff
@@ -27,7 +27,7 @@ public protocol PaymentReviewAPIProtocol: AnyObject {
     func createPaymentRequest(paymentInfo: PaymentInfo, completion: @escaping (Result<String, GiniError>) -> Void)
     func shouldHandleErrorInternally(error: GiniError) -> Bool
     func openPaymentProviderApp(requestId: String, universalLink: String)
-    func submitFeedback(for document: Document, updatedExtractions: [Extraction], completion: @escaping (Result<Void, GiniHealthAPILibrary.GiniError>) -> Void)
+    func submitFeedback(for document: Document, updatedExtractions: [Extraction], completion: ((Result<Void, GiniHealthAPILibrary.GiniError>) -> Void)?)
```

Comment:
Because of a Sonar red issue.
We had a empty clouse on this lines:
`func sendFeedback(updatedExtractions: [Extraction]) { guard let document else { return } `
`delegate?.submitFeedback(for: document, updatedExtractions: updatedExtractions, completion: nil) }`